styles.css

```diff
@@ -1,57 +1,70 @@
-body{
-    width: 375px;
-    align-items: center;
+body {
+  width: 375px;
+  align-items: center;
+  background-repeat: no-repeat;
+  text-align: center;
+}
+
+.heading {
+  background-image: url(../images/header-illsutration-mobile.svg);
 }
 
-nav{
+header {
   background-color: white;
-  margin: 10px
+  margin: 10px;
 }
 
-.header1{
-  display:flex;
+.header1 {
+  display: flex;
   justify-content: space-between;
 }
 
-.header2{
-  display:flex;
+.header2 {
+  display: flex;
   justify-content: space-between;
 }
 
-.menu-button{
+.menu-button {
   width: 40px;
   padding-right: 30px;
   height: 50px;
 }
 
-header{
-  text-align: center;
-  background-image: url(../images/header-illsutration-mobile@2x.png)
+.background-image {
```

Comment:
- [ ] `optional`
- [ ] please move the image to make the words visible as the figma template
![image](https://user-images.githubusercontent.com/66230306/198045381-a0745720-0880-447b-9c7e-3e61139d4535.png)